Kinvey.Core/Auth/User.cs

```diff
@@ -583,69 +583,97 @@ static public async Task LoginWithAuthorizationCodeAPIAsync(string username, str
         /// <param name="micID">[optional] Auth Service ID</param>
         /// <param name="userClient">[optional] Client that the user is logged in, defaulted to SharedClient.</param>
         /// <param name="ct">[optional] CancellationToken used to cancel the request.</param>
+        [Obsolete("This method has been deprecated.  Please use LoginWithMIC() instead.")]
         static public async Task LoginWithMIC(string username, string password, string redirectURI, string micID = null, AbstractClient userClient = null, CancellationToken ct = default(CancellationToken))
```

Comment:
this sounds confusing since you are deprecating a method with the same name. maybe solution would be naming the parameters in your deprecation message like `LoginWithMIC(username:, password:, micID:)`

Reply:
Fixed